### PR TITLE
Change the code style of AADSeleniumITHelper

### PR DESCRIPTION
**Pull request by chenrujun**
Change the code style of AADSeleniumITHelper

### CODE REVIEW

**Comment by chenrujun** [comment]
Closing this PR. the commits will be included in this PR: https://github.com/Azure/azure-sdk-for-java/pull/18373

File: sdk/spring/azure-spring-boot-test-aad/src/test/java/com/azure/test/aad/selenium/AADSeleniumITHelper.java [hunk 1]
```diff
@@ -0,0 +1,115 @@
...
+    public AADSeleniumITHelper(Class<?> appClass, Map<String, String> properties) throws InterruptedException {
+        username = System.getenv(AAD_USER_NAME_1);
+        password = System.getenv(AAD_USER_PASSWORD_1);
+        app = new AppRunner(appClass);
+        DEFAULT_PROPERTIES.forEach(app::property);
+        properties.forEach(app::property);
+
+        ChromeOptions options = new ChromeOptions();
+        options.addArguments("--headless");
+        options.addArguments("--incognito", "--no-sandbox", "--disable-dev-shm-usage");
```
Comment (yiliuTo) [inline sdk/spring/azure-spring-boot-test-aad/src/test/java/com/azure/test/aad/selenium/AADSeleniumITHelper.java:80]: Can these 2 lines be combined?

**Comment by chenrujun** [comment]
Hi, @yiliuTo , please help to review this PR. 🙏

File: sdk/spring/azure-spring-boot-test-aad/src/test/java/com/azure/test/aad/selenium/access/token/scopes/AADAccessTokenScopesIT.java [hunk 2]
```diff
@@ -0,0 +1,95 @@
...
+
+        httpResponse = aadSeleniumITHelper.httpGet("accessTokenScopes/office");
+        Assert.assertFalse(httpResponse.contains("profile"));
+        Assert.assertTrue(httpResponse.contains("https://manage.office.com/ActivityFeed.Read"));
+        Assert.assertTrue(httpResponse.contains("https://manage.office.com/ActivityFeed.ReadDlp"));
+        Assert.assertTrue(httpResponse.contains("https://manage.office.com/ServiceHealth.Read"));
+
+        httpResponse = aadSeleniumITHelper.httpGet("arm");
+        Assert.assertNotEquals(httpResponse, "arm");
+    }
```
Comment (yiliuTo) [inline sdk/spring/azure-spring-boot-test-aad/src/test/java/com/azure/test/aad/selenium/access/token/scopes/AADAccessTokenScopesIT.java:54]: Can we add checks on if the authorization code flow is executed for "arm" client or the same checks as above?

File: sdk/spring/azure-spring-boot-test-aad/src/test/java/com/azure/test/aad/selenium/access/token/scopes/AADAccessTokenScopesIT.java [hunk 3]
```diff
@@ -0,0 +1,95 @@
...
+            return Optional.of(authorizedClient)
+                           .map(OAuth2AuthorizedClient::getAccessToken)
+                           .map(OAuth2AccessToken::getScopes)
+                           .orElse(null);
+        }
+
+        @GetMapping(value = "arm")
+        public String arm(
+            @RegisteredOAuth2AuthorizedClient("arm") OAuth2AuthorizedClient authorizedClient) {
+            return "arm";
```
Comment (yiliuTo) [inline sdk/spring/azure-spring-boot-test-aad/src/test/java/com/azure/test/aad/selenium/access/token/scopes/AADAccessTokenScopesIT.java:91]: Why do we make arm controller different with the others?

**Review by yiliuTo — APPROVED** [review]
LGTM

**Pull request merged**